speaker/bgp_controller.go

```diff
@@ -247,7 +247,7 @@ func (c *bgpController) syncBFDProfiles(profiles map[string]*config.BFDProfile)
 	return c.sessionManager.SyncBFDProfiles(profiles)
 }
 
-func (c *bgpController) SetBalancer(l log.Logger, name string, lbIPs []net.IP, pool *config.Pool) error {
+func (c *bgpController) SetBalancer(l log.Logger, name string, lbIPs []net.IP, pool *config.Pool, client service, svc *v1.Service) error {
```

Comment:
_ service so it's clear we are not using it

Reply:
done